Repeat/repeat.py

```diff
@@ -0,0 +1,56 @@
+
+from PIL import Image
+import numpy as np     # numpy
+from collections import namedtuple
+
+repeat = namedtuple('repeat', 'count pixel')
+
+class Repeat:
+
+	def __init__(self,count1,pixel2):
+		self.r = repeat(count = count1, pixel = pixel2)
+
+#	def count(self):
+#		return self[0]
+#	def pixel(self):
+#		return self[1]
+
+	def generate(self):
+		"""  returns a list repeated pixel_list count number of times """		
```

Comment:
The pixel class should be separate and not inherit from namedtuple. It should have a `def generate` function which returns the individual pixel in a list.